javascript/index.js

```diff
@@ -1,22 +1,306 @@
-const hamburger = document.querySelector('.hamburger');
-const menu = document.querySelector('.nav-list-cont');
-const body = document.querySelector('body');
-const closeModal = document.querySelector('.close-modal');
-const navList = document.querySelectorAll('.nav-list');
+const variables = {
+  hamburger: document.querySelector('.hamburger'),
+  menu: document.querySelector('.nav-list-cont'),
+  body: document.querySelector('body'),
+  closeModal: document.querySelector('.close-modal'),
+  navList: document.querySelectorAll('.nav-list'),
+  sampleContainers: document.querySelector('.sampe-containers'),
+  sampleJobs: document.querySelector('.sample-jobs'),
+  project: document.querySelector('.project'),
+  buttons: document.querySelectorAll('button'),
+  shadow: document.querySelector('.shadow-div'),
+};
 
+// Beginning of navigation pop-up
 function openMenu() {
-  if (menu.classList.toggle('hamburger-toggle', true)) {
-    body.style.overflow = 'hidden';
+  if (variables.menu.classList.toggle('hamburger-toggle', true)) {
+    variables.body.style.overflow = 'hidden';
+    variables.menu.style.overflow = 'visible';
   }
 }
-hamburger.addEventListener('click', openMenu);
+variables.hamburger.addEventListener('click', openMenu);
 
 function closeMenu() {
-  menu.classList.toggle('hamburger-toggle', false);
-  body.style.overflow = 'visible';
+  variables.menu.classList.toggle('hamburger-toggle', false);
+  variables.body.style.overflow = 'visible';
 }
-closeModal.addEventListener('click', closeMenu);
+variables.closeModal.addEventListener('click', closeMenu);
 
-for (let i = 0; i < navList.length; i += 1) {
-  navList[i].addEventListener('click', closeMenu);
+function closeMenuWithNav() {
+  for (let i = 0; i < variables.navList.length; i += 1) {
+    variables.navList[i].addEventListener('click', closeMenu);
+  }
+}
```

Comment:
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```

Reply:
Thanks for your review. But I checked everywhere even the Javascript best practices we were given and this was not a requirement.